packages/tx/src/baseTransaction.ts

```diff
@@ -83,7 +83,17 @@ export abstract class BaseTransaction<TransactionObject> {
   protected DEFAULT_HARDFORK: string | Hardfork = Hardfork.Istanbul
 
   constructor(txData: TxData | AccessListEIP2930TxData | FeeMarketEIP1559TxData) {
-    const { nonce, gasLimit, to, value, data, v, r, s, type } = txData
+    const { data, nonce, gasLimit, gasPrice, r, s, to, type, v, value } = txData
+    this._validateCannotHaveNegativeBN({
```

Comment:
since we run all these values through toBuffer do we need this `_validateCannotHaveNegativeBN` function since you already added the negative check to toBuffer?

Reply:
Indeed! removed.